test/test_scaled_matmul_cuda.py

```diff
@@ -1050,15 +1050,24 @@ def test_scaled_mm_vs_emulated_block_wise_verify_small_shapes(
         # 1x128 blocks need scales to be outer-dim-major
         if lhs_block == 1:
             x_scales = x_scales.t().contiguous().t()
+            lhs_recipe = ScalingType.BlockWise1x128
+        else:
+            lhs_recipe = ScalingType.BlockWise128x128
+
         if rhs_block == 1:
             y_scales = y_scales.t().contiguous().t()
+            rhs_recipe = ScalingType.BlockWise1x128
+        else:
+            rhs_recipe = ScalingType.BlockWise128x128
 
         # Verify that actual F8 mm doesn't error
-        mm_float8(
+        scaled_mm_wrap(
```

Comment:
can we remove `mm_float8`? Honestly not sure if this is still used

Reply:
I've just replaced the last few (missed) uses of this in the test with `scaled_mm_wrap` - if everything still passes ok I'll go ahead and delete.